data_model/src/transaction.rs

```diff
@@ -229,6 +247,143 @@ model! {
         pub payload: TransactionPayload,
     }
 
+    /// Structure that represents the second state of the transaction after receiving at least one signature.
+    ///
+    /// `Iroha` and its clients use [`Transaction`] to send transactions over the network.
+    /// After a transaction is signed and before it can be processed any further,
+    /// the transaction must be accepted by the `Iroha` peer.
+    /// The peer verifies the signatures and checks the limits.
+    #[version(n = 1, versioned = "VersionedSignedTransaction")]
+    #[derive(
+        Debug,
+        Display,
+        Clone,
+        PartialEq,
+        Eq,
+        Hash,
+        Decode,
+        Encode,
+        Deserialize,
+        Serialize,
+        IntoSchema,
+    )]
+    #[display(fmt = "{self:?}")] // TODO ?
```

Comment:
```suggestion
    #[display(fmt = "{}", "self.hash()")]
```

would you fix the display?

Reply:
I can't because there is no `hash()` function without `std` feature enabled. Btw why do we use cryptographic hash for it? Shouldn't it be just a basic non-cryptographic hash just to identify this transaction?

Reply:
but `hash` is in `core::hash`?